src/lib/operators/multi_predicate_join.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "all_type_variant.hpp"
+#include "expression/binary_predicate_expression.hpp"
+#include "expression/evaluation/expression_evaluator.hpp"
+#include "expression/evaluation/expression_result.hpp"
+#include "expression/value_expression.hpp"
+#include "storage/table.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+inline AllTypeVariant _get_value(const Table& table, const RowID row_id, const ColumnID& column_id) {
```

Comment:
`_get_value` is directly within the `opossum` namespace, has a pretty general name and is only used by the other method in this file, `_fulfills_join_predicates`. Also, it is public yet has a leading underscore. You should consider to either rename this and make it public properly or move this into a closer scope.